cli/post-install.js

```diff
@@ -20,7 +20,7 @@ const BINARYEN_SYSTEM_NAME = OS === 'Linux' ? 'Linux' : OS === 'Darwin' ? 'macOS
 const BINARYEN_ARCH_NAME = ARCH === 'x86_64' ? 'X64' : ARCH === 'arm64' ? 'arm64' : 'other';
 const BINARYEN_TAR_NAME = `binaryen-${BINARYEN_SYSTEM_NAME}-${BINARYEN_ARCH_NAME}.tar.gz`;
 
-await executeCommand(`wget https://github.com/near/binaryen/releases/download/${BINARYEN_VERSION_TAG}/${BINARYEN_TAR_NAME}`);
+await executeCommand(`curl https://github.com/near/binaryen/releases/download/${BINARYEN_VERSION_TAG}/${BINARYEN_TAR_NAME}`);
```

Comment:
@thisisjoshford 
curl -L -o  ${BINARYEN_TAR_NAME} https://github.com/near/binaryen/releases/download/${BINARYEN_VERSION_TAG}/${BINARYEN_TAR_NAME}

Reply:
giving that you will need to apply this to all files, maybe you could make an aux method `download`, which creates the command for you.

Reply:
haha was just troubleshooting this issue and pushed a fix then saw this comment.